### PR TITLE
Add fare smoke tests for some deployments

### PR DESCRIPTION
**Pull request by leonardehrenfried**
After the ST fares debacle I'm adding checks that fares are correctly returned to the smoke tests.

This PR includes tests for Portland, Septa and Houston. More will come later when those calculators are fixed.

### CODE REVIEW

File: smoke-tests/portland/router-config.json [hunk 1]
```diff
@@ -3,20 +3,19 @@
...
       "type": "real-time-alerts",
-      "frequencySec": 60,
       "earlyStartSec": 864000,
       "url": "https://trimet.org/transweb/ws/V1/FeedSpecAlerts/includeFuture/true/suppressSystemWideAlerts/true/",
       "feedId": "TriMet"
     },
     {
       "type": "stop-time-updater",
-      "frequencySec": 30,
+      "frequency": "30s",
```
Comment (miles-grant-ibigroup) [inline smoke-tests/portland/router-config.json:12]: I've been seeing this diff all over the place. Is this required now?

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks great! Thanks for this

File: smoke-tests/portland/router-config.json (hunk 1 above)
Comment (leonardehrenfried) [reply smoke-tests/portland/router-config.json:12]: Not required but that's the new syntax. Default is 1 minute.

File: smoke-tests/portland/otp-config.json [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {
   "otpFeatures" : {
-    "SandboxAPILegacyGraphQLApi": true
```
Comment (binh-dam-ibigroup) [inline smoke-tests/portland/otp-config.json:3]: Is this file still needed after this deletion?

File: smoke-tests/septa/otp-config.json [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {
   "otpFeatures" : {
-    "SandboxAPILegacyGraphQLApi": true
```
Comment (binh-dam-ibigroup) [inline smoke-tests/septa/otp-config.json:3]: Is this file still needed after this deletion?

**Review by binh-dam-ibigroup — APPROVED** [review]
Looks fine, but are the empty otp-config.json still needed?
